HW_1/task_1.py

```diff
@@ -0,0 +1,14 @@
+# 1-я задача из домашнего задания №1
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# Пример:
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+n = input("Введите трехзначное число: ")
+n = int(n)
+if (100 <= n <= 999): 
+    a = n % 10
+    b = n % 100 // 10
+    c = n // 100
```

Comment:
Руслан, все верно, но позвольте я немного откорректирую и предложу иной вариант
```suggestion
n = int(input("Введите трехзначное число: "))
if 100 <= n <= 999: 
    a = n % 10
    b = n // 10 % 10
    c = n // 100
```